ui_panels.py

```diff
@@ -1725,6 +1725,27 @@ def invoke(self, context, event):
         return wm.invoke_props_dialog(self, width=500)
 
 
+class OpenSystemFolder(bpy.types.Operator):
```

Comment:
We use term `directory` elsewhere in the add-on, plus the funcs you are adding. I would suggest using the term `directory` in both code and presentation to user instead of folder.

![Screenshot 2024-06-03 at 17 06 22](https://github.com/BlenderKit/BlenderKit/assets/22000268/57b65e9c-1941-4630-8616-3af7f0ee6d99)